mysql-test/suite/rdrs/include/have_rdrs.inc

```diff
@@ -0,0 +1,199 @@
+
+#
+# Check if the needed libs are available
+#
+
+# The expected installation locations are lib, lib/mysql, lib64/mysql, lib/x86_64-linux-gnu
+
+my $F = IO::File->new("$vardir/tmp/have_rdrs_result.inc", 'w') or die;
+print $F "# Looking for rdrs version $version\n";
+
+
+#---------------------------- ndbclient -----------------------------------------
+
+my $ndbclient_lib= my_find_file($bindir,
+                                [ "library_output_directory",
+                                  "lib*/*",
+                                  "lib"
+                                ],
+                                [ "ndbclient.dll",
+                                  "libndbclient.so",
+                                  "libndbclient.dylib"
+                                ],
+                                NOT_REQUIRED);
+
+if ($ndbclient_lib) {
+  my $ndbclient_lib_dir = dirname($ndbclient_lib);
+  print $F "--let \$RDRS_NDBCLIENT_LIB= $ndbclient_lib_dir\n"; 
+  print $F "--echo Found RonDB client library\n"
+} else {
+  # Test runs only when ndb is compiled and enabled.
+  # If not able to find libndbclient, probably an issue with this script.
+  print $F "die Did not find ndbclient library\n";
+}
+
+#---------------------------- rdrclient -----------------------------------------
+
+my $rdrclient_lib = my_find_file($bindir,
+                                [ "library_output_directory",
+                                  "lib*/*",
+                                  "lib"
+                                ],
+                                [ "rdrclient.dll",
+                                  "librdrclient.so",
+                                  "librdrclient.dylib"
+                                ],
+                                NOT_REQUIRED);
+
+if ($rdrclient_lib) {
+   my $rdrclient_lib_dir = dirname($rdrclient_lib);
+   print $F "--let \$RDRS_CLIENT_LIB= $rdrclient_lib_dir\n";
+   print $F "--echo Found RonDB REST server client library\n"
+} else {
+  print $F "--let \$skip_test=1\n";
+  print $F "--let \$skip_reason=This test requires RonDB REST library\n";
+}
+
+#----------------------------- rdrs_string --------------------------------------
+
+my $rdrs_string_lib = my_find_file($bindir,
+                                [ "library_output_directory",
+                                  "lib*/*",
+                                  "lib"
+                                ],
+                                [ "rdrs_string.lib",
+                                  "librdrs_string.a",
+                                  "librdrs_string.a"
+                                ],
+                                NOT_REQUIRED);
+
+if ($rdrs_string_lib) {
+  my $rdrs_string_lib_dir = dirname($ndbclient_lib);
+  print $F "--let \$NDBCLIENT_LIB_DIR= $rdrs_string_lib_dir\n";
+  print $F "--echo Found ndbclient library\n"
+} else {
+  print $F "--let \$skip_test=1\n";
+  print $F "--let \$skip_reason=This test requires rdrs_string library\n";
```

Comment:
die call?

Reply:
same as above, the skip command is written to the have_rdrs_result.inc file. This file is then imported at the end of this script